pkg/handler/data_recorder_kinesis.go

```diff
@@ -0,0 +1,108 @@
+package handler
+
+import (
+	"encoding/json"
+
+	"github.com/checkr/flagr/pkg/config"
+	"github.com/checkr/flagr/pkg/util"
+	"github.com/checkr/flagr/swagger_gen/models"
+
+	"github.com/a8m/kinesis-producer"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kinesis"
+	"github.com/sirupsen/logrus"
+)
+
+type kinesisRecorder struct {
+	enabled  bool
+	producer *producer.Producer
+}
+
+// NewKinesisRecorder creates a new Kinesis recorder
+var NewKinesisRecorder = func() DataRecorder {
```

Comment:
it would be great if we can cover more lines

https://codecov.io/gh/checkr/flagr/pull/151/diff?src=pr&el=tree#diff-cGtnL2hhbmRsZXIvZGF0YV9yZWNvcmRlcl9raW5lc2lzLmdv

Reply:
Yes, it's still WIP, I have to test more things. I just didn't have time yet.